src/pregame.py

```diff
@@ -17,16 +20,19 @@
 from src.cats import Wolfchat, All
 from src import channels
 
-import botconfig
+if TYPE_CHECKING:
+    from src.users import User
+
+import botconfig  # type: ignore
 
 WAIT_LOCK = threading.RLock()
 WAIT_TOKENS = 0
 WAIT_LAST = 0
 
-LAST_START = UserDict() # type: UserDict[users.User, List[datetime, int]]
-LAST_WAIT = UserDict() # type: UserDict[users.User, datetime]
-START_VOTES = UserSet() # type: UserSet[users.User]
-RESTART_TRIES = 0 # type: int
+LAST_START: UserDict[User, List[Union[datetime, int]]] = UserDict()  # FIXME: this should probably be a class containing two fields
```

Comment:
don't add FIXME comments to existing code without consulting the main devs. We do actually care about and track these.